zerver/openapi/zulip.yaml

```diff
@@ -6560,6 +6560,52 @@ paths:
                         "result": "error",
                         "msg": "Saved snippet does not exist.",
                       }
+  /reminders:
+    post:
+      operationId: create-message-reminder
+      tags: ["reminders"]
+      summary: Create a message reminder
+      description: |
+        Create a new reminder for a given message which will be reminded to
+        the current user via a direct message at the specified time.
+
+        **Changes**: New in Zulip 11.0 (feature level 381).
+      requestBody:
+        required: true
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              type: object
+              properties:
+                message_id:
+                  description: |
+                    The ID of the message to be reminded of when the reminder is due.
+                  type: integer
+                  example: 1
```

Comment:
I guess future work would add a `content` field, which would be optional, and when present along with the message ID would be a note, or if present without a message ID would be just the entire text of the reminder DM.

That seems like a pretty simple API change on top of this, so I think this API is good.

Reply:
Okay